tests/test_tng.py

```diff
@@ -24,218 +24,219 @@ def test_open_mode_w(MISSING_FILEPATH):
         pytng.TNGFile(MISSING_FILEPATH, mode='w')
 
 
-def test_open_invalide_mode(GMX_REF_FILEPATH):
+def test_open_invalide_mode(TNG_EXAMPLE):
     with pytest.raises(IOError) as excinfo:
-        pytng.TNGFile(GMX_REF_FILEPATH, mode='invalid')
+        pytng.TNGFile(TNG_EXAMPLE, mode='invalid')
     assert 'mode must be one of "r" or "w"' in str(excinfo.value)
 
 
-def test_len(GMX_REF_DATA, GMX_REF_FILEPATH):
-    with pytng.TNGFile(GMX_REF_FILEPATH) as tng:
-        assert GMX_REF_DATA.length == tng.n_frames
-        assert GMX_REF_DATA.length == len(tng)
+def test_len(TNG_EXAMPLE):
+    with pytng.TNGFile(TNG_EXAMPLE) as tng:
+        assert_equal(tng.n_frames, 10)
```

Comment:
Normal assert is ok for scalar integer comparison... assert_equal is fine, too, but just wanted to note that I dont think that you need to make extra work for you replacing them.

Reply:
okay noted. :)